msu/utilities/math.nut

```diff
@@ -0,0 +1,31 @@
+::MSU.Math <- {
+	// returns log2 of _num, floored (as an int)
+	function log2int( _num )
+	{
+		local count = 0;
+		while (_num > 1) 
+		{
+			_num = _num >> 1;
+			count++;
+		}
+		return count;
+	}
+
+	// Returns the frequency density of _x using normal distribution
+	function normalDistDensity( _x, _mean, _stdev )
+	{
+		local divider = _stdev * sqrt(2 * 3.14);
```

Comment:
perhaps `sqrt(2 * 3.14)` should be saved as a const somewhere and used instead of calculating it in every function call?